### PR TITLE
DOCS Add documentation for configuring a HistoryViewerField for custom DataObjects

### PR DESCRIPTION
**Pull request by robbieaverill**
Issue: https://github.com/silverstripe/silverstripe-versioned-admin/issues/60

### CODE REVIEW

**Comment by NightJar** [comment]
Can't approve, but looks thorough and good :)

File: docs/en/02_Developer_Guides/00_Model/10_Versioning.md [hunk 1]
```diff
@@ -838,7 +838,425 @@ $obj = MyRecord::getComplexObjectRetrieval(); // returns 'Stage' records
 Versioned::set_reading_mode($origMode); // reset current mode
 ```
 
+## Using the history viewer
+
+Since SilverStripe 4.3 you can use the React and GraphQL driven history viewer UI to display historic changes and
+comparisons for a versioned DataObject. This is automatically enabled for SiteTree objects and content blocks in
```
[the remark this answers is not in the document]
Comment (ScopeyNZ) [reply docs/en/02_Developer_Guides/00_Model/10_Versioning.md]: This doesn't matter in markdown 😴

File: docs/en/02_Developer_Guides/00_Model/10_Versioning.md [hunk 2]
```diff
@@ -838,7 +838,425 @@ $obj = MyRecord::getComplexObjectRetrieval(); // returns 'Stage' records
...
+    };
+  },
+};
+
+export { query, config };
+
+export default graphql(query, config);
+```
+
+**app/client/src/state/revertToMyVersionedObjectVersionMutation.js**
```
[the remark this answers is not in the document]
Comment (ScopeyNZ) [reply docs/en/02_Developer_Guides/00_Model/10_Versioning.md]: We can follow up in the applicable branch once this is merged 🙂

File: docs/en/02_Developer_Guides/00_Model/10_Versioning.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (robbieaverill) [reply docs/en/02_Developer_Guides/00_Model/10_Versioning.md]: Yeah there’s a difference between devs- some use 80 chars others 120, sometimes none at all. I don’t think we have formalised anything yet

**Pull request merged**